
class Header():
    def __init__(self, robotCorpId=None, workCorpId=None, workSiteId=None, robotId=None, robotType=None):
        self.robotCorpId = robotCorpId
        self.workCorpId = workCorpId
        self.workSiteId = workSiteId
        self.robotId = robotId
        self.robotType = robotType